src/cli/commands/graph.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+import {Command, flags} from "@oclif/command";
+import mkdirp from "mkdirp";
+import os from "os";
+import path from "path";
+
+import {pluginNames} from "./common";
+
+const execDependencyGraph = require("../../tools/execDependencyGraph").default;
+
+export default class GraphCommand extends Command {
+  static description = "create the contribution graph for a repository";
+
+  static args = [
+    {
+      name: "repo_owner",
+      required: true,
+      description: "owner of the GitHub repository for which to fetch data",
+    },
+    {
+      name: "repo_name",
+      required: true,
+      description: "name of the GitHub repository for which to fetch data",
+    },
+  ];
+
+  static flags = {
+    "output-directory": flags.string({
+      short: "o",
+      description: "directory into which to store graphs",
+      env: "SOURCECRED_OUTPUT_DIRECTORY",
+      default: path.join(os.tmpdir(), "sourcecred"),
+    }),
+    "github-token": flags.string({
+      description:
+        "a GitHub API token, as generated at " +
+        "https://github.com/settings/tokens/new",
+      required: true,
+      env: "SOURCECRED_GITHUB_TOKEN",
+    }),
+  };
+
+  async run() {
+    const {
+      args: {repo_owner: repoOwner, repo_name: repoName},
+      flags: {"github-token": token, "output-directory": outputDirectory},
```

Comment:
this is cool btw, I didn't realize destructuring assignment could be used this way. I'm really a fan of destructuring assignment now, it can make for some really clean code.